fuel/transformers/__init__.py

```diff
@@ -97,8 +113,8 @@ def get_data(self, request=None):
 
 class ForceFloatX(Transformer):
     """Force all floating point numpy arrays to be floatX."""
-    def __init__(self, data_stream):
-        super(ForceFloatX, self).__init__(data_stream)
+    def __init__(self, data_stream, batch=False):
```

Comment:
Same as above. It's probably better if it crashes when you try to pass `batch` anyway, because it won't have any effect.